src/main/java/com/ace/alfox/game/MoveAction.java

```diff
@@ -34,6 +38,11 @@ public ActionResult applyAction(Player player, Map<String, Object> params) {
                                     // maybe do running away mechanic here?
         player.location = player.location.add(directions.get(direction));
         result.log("traveled " + direction);
+
+        Location location = locationFactory.fetch(player.location);
+        result.log("You see, " + location.title)
+              .log(location.description);
+

```

Comment:
I wanted to add in a DI data accessor as an example of how it can be used in the Actions